src/Api/Operator/Certificate.php

```diff
@@ -21,19 +21,34 @@ public function generate(array $properties): Struct\Info
         return new Struct\Info($response);
     }
 
-    public function install(array $properties, string $request, string $privateKey): bool
+    public function install(array $properties, $request, string $privateKey = null): bool
```

Comment:
`?string $privateKey` - because it is nullable. or just remove type here, because it is legacy.